docs/pages/guides/monorepos.md

```diff
@@ -190,6 +260,39 @@ export default function App() {
 
 As mentioned earlier, using monorepos is not for everyone. You take on increased complexity and need to solve issues you most likely will run into. Here are a couple of common issues you might encounter.
 
+### Can I use another monorepo tool instead of yarn workspaces?
+
+There are a lot of monorepo tools available, and each of these tools has its benefits. It's hard for us to keep up with the latest tools and methods, and because of that, we won't officially support new monorepo tools. That being said, if the tool follows these three rules, it should work fine.
```

Comment:
```suggestion
There are a lot of monorepo tools available, and each of these tools has its benefits. It's hard for us to keep up with the latest tools and methods, and because of that, we can't officially support new monorepo tools. That being said, if the tool follows these three rules, it should work fine.
```